test/test_all.py

```diff
@@ -521,6 +521,10 @@ class ExampleExpression(Example):
                               Example.get_sample("simple_test.bin")],
                              products=["graph_instr.dot", "out.dot"])
 
+testset += ExampleExpression(["access_c.py", Example.get_sample("human.bin")],
```

Comment:
In the next commit, `pycparser` is noted as optional (which seems a good idea). To reflect this, you could add a tag, as for `z3`, which is automatically disabled if `pycparser` is not found on the machine. Otherwise, one cannot launch tests without this dependency...